android/build.gradle

```diff
@@ -16,10 +20,12 @@ android {
     }
 }
 
+def supportLibVersion = rootProject.hasProperty('supportLibVersion') ? rootProject.supportLibVersion : SUPPORT_LIB_VERSION
+
 dependencies {
-    compile 'com.android.support:appcompat-v7:27.0.2'
-    compile 'com.facebook.react:react-native:+' // support react-native-v0.22-rc+
-    compile 'com.facebook.android:facebook-android-sdk:4.+'
+    implementation "com.android.support:appcompat-v7:${safeExtGet('supportLibVersion', '27.0.2')}"
+    api "com.facebook.react:react-native:+" // support react-native-v0.22-rc+
+    api "com.facebook.android:facebook-android-sdk:4.+"
```

Comment:
This line has a conflict with the `master` branch.

Reply:
I opened https://github.com/facebook/react-native-fbsdk/pull/394 to extract what was not on master yet from this PR. Could you please release a new version to NPM with all the last fixes please once you merged it :) ?